OrangeHRMTests/PageObjects/Modules/LeftMenuNavigationPanel.cs

```diff
@@ -0,0 +1,38 @@
+﻿using OpenQA.Selenium;
+using OrangeHRMTests.Common.Drivers;
+using OrangeHRMTests.Common.WebElements;
+
+namespace OrangeHRMTests.PageObjects.Modules
+{
+    public class LeftMenuNavigationPanel
+    {
+        private MyWebElement AdminMenuItem = new MyWebElement(By.XPath("//a[@href='/web/index.php/admin/viewAdminModule']"));
```

Comment:
a[@href='/web/index.php/ - for many locators it's a common part

Reply:
Fixed